api/users/serializers.py

```diff
@@ -102,6 +102,8 @@ def update(self, instance, validated_data):
                 setattr(instance, attr, value)
         try:
             instance.save()
+        except ValidationError as e:
+            raise InvalidModelValueError(detail=e.args)
```

Comment:
I can do that, what I had planned to do was...
```
if 'social' in e.args[0]:
    raise InvalidModelValueError(detail=e.args[0]['social'][0][0])
```
That way only the message comes trough. However, I see in the production sentry errors that most ValidationErrors are using e.args[0] . What do you think? Want I should just go with e.args[0] ?